src/main/java/randoop/main/GenTests.java

```diff
@@ -293,6 +299,24 @@ public boolean handle(String[] args) {
       classNameErrorHandler = new WarnOnBadClassName();
     }
 
+    if (test_add_dependencies) {
+      // The first two lines are in this order to avoid having getDependentClassnamesFromClassnames
+      // act on classes added by getDependentClassnamesFromMethodList.
+      classnames.addAll(getDependentClassnamesFromClassnames(classnames, accessibility));
+      classnames.addAll(getDependentClassnamesFromMethodList(accessibility, omit_methods));
```

Comment:
It is inconsistent that `getDependentClassnamesFromClassnames` takes `classnames` as an argument, but `getDependentClassnamesFromMethodList` uses a field.  Method implementations and uses are easiest to understand when they do not use external state.  Could you change `getDependentClassnamesFromMethodList`?